app/src/androidTest/java/org/fnives/test/showcase/ui/ComposeLoginRobot.kt

```diff
@@ -0,0 +1,53 @@
+package org.fnives.test.showcase.ui
+
+import android.content.Context
+import androidx.compose.ui.test.assertCountEquals
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.assertTextContains
+import androidx.compose.ui.test.junit4.ComposeTestRule
+import androidx.compose.ui.test.onAllNodesWithTag
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.performClick
+import androidx.compose.ui.test.performTextInput
+import androidx.test.core.app.ApplicationProvider
+import org.fnives.test.showcase.compose.screen.auth.AuthScreenTag
+
+class ComposeLoginRobot(
```

Comment:
I see that basically every method needs the ComposeTestRule.
What do you think if we were doing something like this:

`class ComposeLoginRobot(composeTestRule: ComposeTestRule) : ComposeTestRule by composeTestRule`

That way all composeTestRule references could be removed. I know we would be exposing the Interface's methods as well, but since that's where we are getting the TestRule itself, I don't think that's too big of an issue, specially since it's just in test code

Reply:
I wouldn't have thought to do this. Yes, every method needs the rule. I don't think it is a problem to expose `ComposeTestRule`.